src/mods.cpp

```diff
@@ -80,12 +80,15 @@ void parseModContents(ModSpec &spec)
 	}
 }
 
-std::map<std::string, ModSpec> getModsInPath(std::string path, bool part_of_modpack)
+std::map<std::string, ModSpec> getModsInPath(std::string path, bool part_of_modpack, std::string worldpath)
```

Comment:
> I'd love to see warnings during compilation... by default.

Some warnings are enabled, some are disabled.  What does that have to do with this though?